hack/start_flotta.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+start_operator() {
+  read -p "Enter operator path: " OPERATOR_PATH
```

Comment:
This force the user to manually insert every time all the required paths. IMO it's not the best.

Reply:
So I split the script into three scripts :) each will receive different arguments